packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -241,6 +249,13 @@ class Auth {
 
         this.redirectURI = config.redirectURI
 
+        this.passwordlessConfig = {
+            ...config.passwordlessConfig,
+            ...(callbackURI && {
+                callbackURI: isAbsoluteUrl(callbackURI) ? callbackURI : `${baseUrl}${callbackURI}`
```

Comment:
For the scenario where the callbackURI isn't absolute, do we know if we are going to be composing the value correctly.. or is there a potential to have something like `http://www.test.com//mypath` or `http://www.test.commypath` depending on what the baseURL is and the callbackURI?

Reply:
I am updating the `callbackURI` type to look like this:
`callbackURI?: `/${string}` | `http://${string}` | `https://${string}``
Therefore, in the case we have a valid relative path, I think we should depend on these type restrictions to build a valid URL.